docs/ZendeskMultiObjects-batchsource.md

```diff
@@ -85,16 +89,23 @@ corresponding CDAP types.
 | Array                  | Array     |
 | Record                 | Record    |
 
-Data Duplication issues with Zendesk APIs
+Limitations
 ----------
-This plugin can return duplicate records for time-based exports as mentioned in Zendesk docs.
+
+Zendesk supports three types of pagination: [offset](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-offset-pagination/), [time-based](https://developer.zendesk.com/documentation/ticketing/managing-tickets/using-the-incremental-export-api/#time-based-incremental-exports)  pagination.
```

Comment:
change to "two types". and "and" rather than a comma between the two types

Reply:
done